CMakeLists.txt

```diff
@@ -39,9 +39,9 @@ set(OVS_SOURCE_DIR "${CMAKE_SOURCE_DIR}/ovs/ovs" CACHE PATH
 set(SDE_INSTALL_DIR "$ENV{SDE_INSTALL}" CACHE PATH
     "SDE install directory")
 
-set(PROTO_OUT_DIR "${CMAKE_BINARY_DIR}/pb.out" CACHE PATH
+set(PB_OUT_DIR "${CMAKE_BINARY_DIR}/pb.out" CACHE PATH
```

Comment:
Would `PROTOBUF_OUT_DIR` be a better name?

Reply:
`PB_OUT_DIR` tracks the directory name, which is `pb.out`.

There are two kinds of Protobuf files: files written in the Protobuf interface language, which have the file type `.proto`, and language-specific files generated by the Protobuf compiler. For C++, these files have file types `.pb.cc` and `.pb.h`. The files written to the `pb.out` directory are the latter.

`proto` refers to files that are inputs to the Protobuf compiler. `pb` refers to files that are outputs of the Protobuf compiler.

Reply:
Ack.
Rest looks good.